doc/classes/AnimationTreePlayer.xml

```diff
@@ -45,6 +46,7 @@
 			<argument index="0" name="id" type="String">
 			</argument>
 			<description>
+				Returns the name of the [member master_player]'s animation bound to this animation node.
```

Comment:
not sure if 'bound' is the best word, maybe "animation that's binded to this animation node"

Reply:
'bound' is the past tense of 'bind'. Bind is an irregular verb

Reply:
@Zireael07 oh, sorry! Thank you for the info, I'm still struggling with English.. :( 
:smile: